package.json

```diff
@@ -112,7 +122,8 @@
     "temp": "^0.9.0",
     "through": "^2.3.8",
     "webpack": "^4.29.3",
-    "webpack-cli": "^3.2.3"
+    "webpack-cli": "^3.2.3",
+    "yorkie": "^2.0.0"
```

Comment:
we can consider using `husky` -- it seems to be more popular?

(I'm not very familar with `yorkie` (o|o)

Reply:
`yorkie` is just a fork of `husky`.

Reply:
`yorkie` takes care of some of the problems `husky` has with resolving the correct location of `package.json` (Vue uses `yorkie`).